### PR TITLE
Fix contact section

### PR DESCRIPTION
**Pull request by Leboroz**
I changed the div for a form tag and wrapped every field with a li tag

### CODE REVIEW

File: views/index.html [hunk 1]
```diff
@@ -171,28 +171,40 @@ <h2>Get started</h2>
...
+          <label id="message-label" for="message">Message</label>
+          <textarea
+            id="message"
+            required
+            placeholder="Enter your message here..."
+          >
+          </textarea>
+        </li>
+          </ul>
+        </form>
```
Comment (safafa) [inline views/index.html:206]: - [ ] The form must have a submit button. You can use a button with the type submit or an input type submit. I suggest moving the ` Start collaboration` button inside the form and adding `type="submit"` attribute to it.

File: views/index.html [hunk 2]
```diff
@@ -171,28 +171,40 @@ <h2>Get started</h2>
...
-        <input
-          class="field"
-          id="username"
-          type="text"
-          required
-          placeholder="name"
-        />
-        <input
+        <form action="/contact" method="POST">
+          <ul>
```
Comment (safafa) [inline views/index.html:175]: image

- [ ] As requested in the project requirements You must collect data using Formspree. Kindly make the change to made the form functional.

**Review by safafa — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED ♻️ 

Hi @Leboroz  @alvinlouis29 , You did a great job so far 👏🏼 👏🏼 

- Well done on following the original design.

However, here are some points that you need to work on to get your project approved.

## Required changes :

- [ ] Check the inline comments.

## Optional changes :
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: views/index.html (hunk 2 above)
Comment (Leboroz) [reply views/index.html:175]: Thank you for your feedback! 🤟

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Leo Albornoz Cubillán, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏

## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**